files/windows/alias.psm1

```diff
@@ -0,0 +1,21 @@
+## Name: C:\Users\Administrator\Documents\WindowsPowerShell\Modules\alias\alias.psm1
+
+Function validate_erb_func {
+ [CmdletBinding()]
+   Param($arg)
+   $pwd = Get-Location
+   $filename = "$pwd\$arg"
+   $command = 'erb -P -x -T - $filename | ruby -c'
+   iex "& $command"
+}; Set-Alias validate_erb validate_erb_func

```

Comment:
First powershell script - was just working off examples. If the Set-Alias isn't required it can certainly be removed.